docs/support.md

```diff
@@ -3,17 +3,66 @@ hide:
-Our support team will answer it by replying or updating documentation.
+Our support team will answer it by replying to the issue or by updating documentation.
 
-## Migration Assistance
+# Priority Support
 
-Cirrus Labs can help your team with migration to Cirrus CI. Our team will analyze your current needs and workflow in order
-to not only perform the migration, but also perform optimizations along the way to make sure your team's workflow is as 
-optimal as it can possibly be.
+In addition to the general support we provide a *Priority Support* option with guaranteed response times and some extra
+care like bi-yearly checkins to make sure Cirrus CI roadmap is aligned with your company's needs.
 
-Our team has experience of optimizing developer experiences at companies like Airbnb and Twitter. Contact us at [support+migration@cirruslabs.org](mailto:support+migration@cirruslabs.org).
+| Severity | Support Impact                                                                               | First Response Time SLA | Hours | How to Submit                     |
+|----------|----------------------------------------------------------------------------------------------|-------------------------|-------|-----------------------------------|
+| 1        | Emergency (Completely unusable)                                                              | 30 minutes              | 24x7  | Please trigger an urgent issue    |
+| 2        | Highly Degraded (Important features unavailable or extremely slow; No acceptable workaround) | 4 hours                 | 24x5  | Please use priority email address |
+| 3        | Medium Impact                                                                                | 8 hours                 | 24x5  | Please use priority email address |
+| 4        | Low Impact                                                                                   | 24 hours                | 24x5  | Please use priority email address |
+
+`24x5` means period of time from 9AM on Monday till 5PM on Friday in EST timezone.
+
+<!-- markdownlint-disable MD037 -->
+??? note "Support Impact Definitions"
+    * **Severity 4** - Questions or Clarifications around features or documentation (24 hours) Minimal or no Business Impact. 
+      Information, an enhancement, or documentation clarification is requested, but there is no impact on the operation of Cirrus CI.
+    * **Severity 3** - Something is preventing normal Cirrus CI operation (8 hours) Some Business Impact. Important Cirrus CI
+      features are unavailable or somewhat slowed, but a workaround is available. Cirrus CI use has a minor loss of operational functionality.
+    * **Severity 2** - Cirrus CI is Highly Degraded (4 hours) Significant Business Impact. Important Cirrus CI features are unavailable
+      or extremely slowed, with no acceptable workaround.
+    * **Severity 1** - Cirrus CI is unavailable or completely unusable (30 Minutes). An urgent issue can be filed and
```

Comment:
Consider applying DRY to this section (`Support Impact Definitions`):

* skip things like `(4 hours)` and `(30 Minutes)` as they're already documented in the table above
* move short impact explanations (`Cirrus CI is unavailable or completely unusable`) to the table above and only keep the full descriptions here

This will improve reading experience as the user no longer needs to skip over the already learned details.